docs/BindingsGuide_DotnetOutOfProc.md

```diff
@@ -0,0 +1,187 @@
+
+## Binding Model
+
+.NET Isolated introduces a new binding model, slightly different from the binding model exposed in .NET Core 3 Azure Functions. More information can be [found here](https://github.com/Azure/azure-functions-dotnet-worker/wiki/.NET-Worker-bindings). Please review our samples for usage information.
+
+## Samples
+
+You can find samples on how to use Sql Extension .NET Worker under `samples\samples-outofproc` ([link](https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-outofproc)).
+
+## Create and run .NET Isolated functions
+
+**Note: Visual Studio and Visual Studio Code support is on the way. In the meantime, please use `azure-functions-core-tools` or the sample projects as a starting point.**  
+
+### Install .NET 6.0
+
+Download .NET 6.0 [from here](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+### Install the Azure Functions Core Tools
+
+To download Core Tools, please check out our docs at [Azure Functions Core Tools](https://github.com/Azure/azure-functions-core-tools)
+
+### Create an Isolated Function App
+
+Run the below in the Terminal/CLI or with PowerShell.
+
+    ```
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime dotnet (Isolated Process)
+    ```
+
+   or in an empty directory, run `func init` and select `dotnet (Isolated Process)`
+
+### Add a function: 
+Run `func new` and select `HttpTrigger` trigger. Fill in the function name.
+
+### Configure Function App
+
+1. Get your SQL connection string
+    
+    Local SQL Server - Use this connection string, replacing the placeholder values for the database and password.  
+      
+    `Server=localhost;Initial Catalog={db_name};Persist Security Info=False;User ID=sa;Password={your_password};` Azure SQL Server - Browse to the SQL Database resource in the [Azure portal](https://ms.portal.azure.com/)  
+    \- In the left blade click on the **Connection Strings** tab  
+    \- Copy the **SQL Authentication** connection string  
+      
+    (_Note: when pasting in the connection string, you will need to replace part of the connection string where it says '{your\_password}' with your Azure SQL Server password_)
+2. Open the generated `local.settings.json` file and in the `Values` section verify you have the below. If not, add the below and replace `{connection_string}` with the your connection string from the previous step:
+    
+    ```
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+    "SqlConnectionString": "{connection_string}"
+    
+    ```
+    
+3. Verify your `host.json` looks like the below:
```

Comment:
Do we need this? There isn't anything special in here that we need to modify so I'd say just get rid of this and let the default generated be used